build/storage/tests/it/ptf_tests/system_tools/test_platform.py

```diff
@@ -255,6 +255,24 @@ def create_virtio_blk_devices(
             )
         return device_handles
 
+    def create_fake_virtio_blk_devices(
```

Comment:
It is not fake virtio-blk device. Here you are creating virtio-blk devices with invalid device handles, aren't you?

Reply:
I want to not create virtio blk, I want delete non exist virtio_blk_device but now we have dedicated class so I create class but this class not create virtio_blk -

Reply:
I understand. They are not fake. Fake word is [associated ](https://testing.googleblog.com/2013/07/testing-on-toilet-know-your-test-doubles.html)with test doubles and it provides a lightweight implementation of an API that behaves like the real implementation, but isn't suitable for production.
Here you need just test what happens with invalid device handle. No device creation is performed.
So, would it be even easier to write your test like:
```
virtio_blk_with_invalid_device_handle = VirtioBlkDevice(
        f"some_invalid_device_handle_{volume}_{physical_id}",
        volume,
        self.ipu_storage_platform,
        host_target_address_service,
    )
delete_result = self.ipu_storage_platform.delete_virtio_blk_device(virtio_blk_with_invalid_device_handle)
self.assertFalse(delete_result )
```

Reply:
deleted